### DISCUSSION TITLE
Support for new SEB Windows and MacOS versions

### DISCUSSION THREAD

**Discussion by naman912** [SafeExamBrowser/seb-win-refactoring#1067] [General]
Hi, some of our customers use SEB. Is there a mailing list/RSS feed etc to subscribe to get notified about new Windows and Mac versions whenever they are released by SEB? 
We want to automate the creation of Support tickets internally in our org as and when new SEB Windows and Mac versions are released .
Ex: New SEB version for windows needs to be tested on recent Windows OS to offer support of our assessments

Also is there a pre release version?

**Comment by danschlet** [comment]
Best if you add notifications on the [Releases](https://github.com/SafeExamBrowser/seb-win-refactoring/releases) pages in this SEB for Windows repository and the [macOS repository](https://github.com/SafeExamBrowser/seb-mac/releases) to be notified about new releases. Pre-release versions for macOS are also available on that page, for Windows go to https://sebdev.ethz.ch/project/appveyor/seb-win-refactoring, select x86 and Artifacts and download the .exe installer bundle (containing both x64 and x86 versions) of the most up-to-date development build. For iOS there is a [public TestFlight link](https://testflight.apple.com/join/egxLmc4j).

There is also an [RSS feed](https://sourceforge.net/p/seb/news/feed.rss) (as stated on the website!), but we sometimes don't have the capacity to update it immediately. We don't anymore post updates on the eX-Twitter account.
